source/browserapis/animation.html.md

```diff
@@ -10,11 +10,24 @@ mdn_docs: https://developer.mozilla.org/en-US/docs/Web/API/Animation
 web_platform: https://docs.webplatform.org/wiki/apis/web_animations
 
 links_tutsarts:
+  'Web Animations API Tutorial Part 1: Creating a Basic Animation' : http://danielcwilson.com/blog/2015/07/animations-part-1/
+  'Web Animations API Tutorial Part 2: The AnimationPlayer & Timeline Controls' : http://danielcwilson.com/blog/2015/07/animations-part-2/
+  'Web Animations API Tutorial Part 3: Multiple Animations' : http://danielcwilson.com/blog/2015/08/animations-part-3/
+  'Web Animations API Tutorial Part 4: GroupEffects & SequenceEffects' : http://danielcwilson.com/blog/2015/09/animations-part-4/
+  'Web Animations API Tutorial Part 5: The Loveable Motion Path' : http://danielcwilson.com/blog/2015/09/animations-part-5/
+  'Web Animations API Tutorial Conclusion' : http://danielcwilson.com/blog/2015/09/animations-conclusion/
```

Comment:
I think so @andrew, from looking at it I think that GitHub syntax highlighting is getting confused by their being : inside the quotes,